common/models/model-method.js

```diff
@@ -10,4 +10,43 @@ module.exports = function(ModelMethod) {
    * @class ModelMethod
    * @inherits WorkspaceEntity
    */
+
+  ModelMethod.getJsonKey = function(data) {
+    var version = ModelMethod.app.models.Workspace.loopBackVersion;
+    if (encodeStaticFlagInName(version)) {
+      var isStatic = data.isStatic;
+      if (isStatic !== undefined) {
+        var name = data.name;
+        var matchName = name.match(/^prototype\.(.*)$/);
+        if (!isStatic && (matchName === null || !matchName)) {
+          data.name = 'prototype.' + name;
+        }
+      }
+      delete data.isStatic;
+    }
+    return data.name;
+  }
+
+  ModelMethod.getConfigFromData = function(data) {
+    var version = ModelMethod.app.models.Workspace.loopBackVersion;
+    var properties = this.definition.properties;
+    var result = {};
+    var prop;
+
+    if (encodeStaticFlagInName(version)) {
+      delete data.isStatic;
+    }
+    
+    for (prop in properties) {
+      if (properties[prop].json === false) continue;
+      result[properties[prop].json || prop] = data[prop];
+    }
```

Comment:
Please call parent (super) method instead of duplicating this code. Example:

```js
ModelMethod.getConfigFromData = function(data) {
  var config = ModelMethod.base.getConfigFromData(data);
  if (encodeStaticFlagInName(version)) {
    delete config.isStatic;
  }
  return config;
}
```